frappe/erpnext
# fix: Zero rated exports in GSTR-3B report

Export Invoices without payment of taxes were not showing up in GSTR-3B report

<img width="1198" alt="Screenshot 2022-02-06 at 7 04 33 PM" src="https://user-images.githubusercontent.com/42651287/152683465-9e4a4281-33af-412b-90ef-3680b1524449.png">

Steps to test:
1. Make an overseas customer with GST Category as "Overseas" and "Export Type" as "Without Payment Of Tax"
2. Create a GST Sales Invoice against this customer, taxes won't be applied since the export type is without payment of tax
3. Generate GSTR-3B report for this month and check if the taxable value show up as seen above
